src/Compilers/CSharp/Portable/Symbols/MemberSymbolExtensions.cs

```diff
@@ -453,7 +454,7 @@ internal static bool IsParameterlessConstructor(this MethodSymbol method)
         internal static bool IsDefaultValueTypeConstructor(this MethodSymbol method, bool requireZeroInit)
         {
             if (method.IsImplicitlyDeclared &&
-                method.ContainingType.IsValueType &&
+                method.ContainingType?.IsValueType == true &&
```

Comment:
>method.ContainingType?.IsValueType == true [](http://example.com/codeflow?start=16&length=42)

Any idea why this change wasn't needed for "ComputeStringHash" previously?

Reply:
This is the first time we are using "synthesized global methods" in the bound tree.￼